index.html

```diff
@@ -0,0 +1,87 @@
+                </tr>
+                <tr>
+                    <td><img src="https://media-exp1.licdn.com/dms/image/C560BAQE1H3S3iDNY6Q/company-logo_200_200/0/1569251131340?e=2159024400&v=beta&t=2VEZRmX14hm_7jPA58UKXAaKFf_nwhSqWh135Irkte8" alt=""></td>
+                    
+                    <td><a href="https://yahoo.com" target="_blank" class="engines">Yahoo</a></td>
+                </tr>
+                <tr>
+                    <td><img src="https://pbs.twimg.com/profile_images/1373867336021598212/vZ2nWy8M.jpg" alt=""></td>
+                    
+                    <td><a href="https://baidu.com" target="_blank" class="engines">Baidu</a></td>
+                </tr>
+                <tr>
+                    <td><img src="https://e7.pngegg.com/pngimages/124/307/png-clipart-duckduckgo-web-search-engine-google-search-anonymity-eyebrows-miscellaneous-logo.png" alt=""></td>
+                    
+                    <td><a href="https://duckduckgo.com/" target="_blank" class="engines">DuckDuckGo</a></td>
+                </tr>
+                
+            </table>
+        </div>
+        
+        <div class="right-container">
+            <h2>Recent Visited Websites</h2>
+            <table class="right-table">
+                <tr>
+                    <td><img src="https://yt3.ggpht.com/584JjRp5QMuKbyduM_2k5RlXFqHJtQ0qLIPZpwbUjMJmgzZngHcam5JMuZQxyzGMV5ljwJRl0Q=s900-c-k-c0x00ffffff-no-rj" alt=""></td>
+                    
+                    <td><a href="https://youtube.com" target="_blank" class="engines">YouTube</a></td>
+                </tr>
+                <tr>
+                    <td><img src="https://leetcode.com/static/images/LeetCode_logo_rvs.png" alt=""></td>
+                    
+                    <td><a href="https://leetcode.com" target="_blank" class="engines">Leetcode</a></td>
+                </tr>
+                <tr>
+                    <td><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS88pVvJZYLv-4KMAY4CxgLyk7yoCxcccizmkC6J5Xe17E4Y6O1cHfTstN5CA1tCNXds-w&usqp=CAU" alt=""></td>
+                    
+                    <td><a href="https://W3Schools.com" target="_blank" class="engines">W3Schools</a></td>
+                </tr>
+                <tr>
+                    <td><img src="https://media.geeksforgeeks.org/wp-content/cdn-uploads/gfg_200x200-min.png" alt=""></td>
+                    
+                    <td><a href="https://gfg.com" target="_blank" class="engines">GeeksForGeeks</a></td>
+                </tr>
+                <tr>
+                    <td><img src="https://image.shutterstock.com/image-photo/kiev-ukraine-june-7-2015-260nw-285114926.jpg" alt=""></td>
+                    
+                    <td><a href="https://www.linkedin.com/in/mukesh12/" target="_blank" class="engines">LinkedIn</a></td>
+                </tr>
+                
+            </table>
+        </div>
+    </div>
+</body>
+</html>
```

Comment:
all good, the webpage looks really good and complete!

Reply:
Thanks Rijav